### PR TITLE
update: alerts

### PR DESCRIPTION
**Pull request by rkapoor10**
added alerts and please ignore the code section...it will be added later. Facing some issues with carbon. 
Thanks for reviewing. 😄

### CODE REVIEW

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    border-radius: 4px;
+  }
+  
+  .snackbar-btn-style:active {
+    opacity: 0.4;
+    background-color: var(--lightorchid-color);
+    color: var(--darkorchid-color);
+  }
+  
+  .fas {
```
Comment (JantuDeb) [inline components/alert/alert.css:144]: Avoid using classes from external library use your own class or ` .parentclass element {}` similar structure

**Review by JantuDeb — APPROVED** [review]
Alert  component can be marged.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,157 @@
+.alert{
+
+    justify-content: center;
+    align-items: center;
+    justify-content: space-evenly;
+    margin: 2rem;
+    border: 2px solid;
+    justify-content: center;
```
Comment (harshitadaswani) [inline components/alert/alert.css:8]: justify-content property used 3 times

**Review by harshitadaswani — CHANGES_REQUESTED** [review]

File: components/alert/alert.css (hunk 1 above)
Comment (rkapoor10) [reply components/alert/alert.css:144]: Thanks...will keep that in mind and make changes.

File: components/alert/alert.css (hunk 2 above)
Comment (rkapoor10) [reply components/alert/alert.css:8]: Oh! Thanks will make the changes.

**Comment by hussiantabha** [comment]
Looks good Ritik

**Pull request merged**